### PR TITLE
Code Hygiene - glog to klog

### PR DESCRIPTION
**Pull request by chrislovecnm**
Cleaning house a bit and changing the work 'glog' to 'klog' in
a few places.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

File: klog_test.go [hunk 1]
```diff
@@ -473,7 +473,7 @@ func TestLogBacktraceAt(t *testing.T) {
 		// Need 2 appearances, one in the log header and one in the trace:
 		//   log_test.go:281: I0511 16:36:06.952398 02238 log_test.go:280] we want a stack trace here
 		//   ...
-		//   github.com/glog/glog_test.go:280 (0x41ba91)
+		//   github.com/kubernetes/klog/klog_test.go:280 (0x41ba91)
```
Comment (dims) [inline klog_test.go]: `github.com/kubernetes/klog/klog_test.go` should be`k8s.io/klog/klog_test.go`. thanks!

Comment (chrislovecnm) [reply klog_test.go]: Is the correct line number?

Comment (andyxning) [reply klog_test.go]: Line number does not matter, imo. But the package name matters.

@chrislovecnm Package name should be corrected. :)

Comment (andyxning) [reply klog_test.go]: Others lgtm.

Comment (chrislovecnm) [reply klog_test.go]: @dims fixed

**Comment by chrislovecnm** [comment]
@detiber @dims PTAL

**Pull request merged**